tests/test_table_transformer.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from unittest import TestCase
+
+import habana_frameworks.torch as ht
+import pytest
+import torch
+from huggingface_hub import hf_hub_download
+from PIL import Image
+from transformers import AutoImageProcessor, TableTransformerForObjectDetection
+
+from optimum.habana.transformers.modeling_utils import adapt_transformers_to_gaudi
+
+
+adapt_transformers_to_gaudi()
+
+LATENCY_TABLE_TRANSFORMER_BF16_GRAPH_BASELINE = 2.2
```

Comment:
this appears to be the value for Gaudi2. Correct?
Have you tested this on Gaudi1?

Reply:
This is the value for Gaudi2. I have not tested on Gaudi1 since I do not have access to a Gaudi1 machine. Could you add that part for me? Thanks!